libs/core/src/schemas/commands/subscription.schemas.ts

```diff
@@ -0,0 +1,67 @@
+import { z } from 'zod';
+import {
+  CommentSubscription,
+  CommunityAlert,
+  SubscriptionPreference,
+  ThreadSubscription,
+} from '../entities.schemas';
+
+export const UpdateSubscriptionPreferences = {
+  input: z.object({
+    email_notifications_enabled: z.boolean().optional(),
+    digest_email_enabled: z.boolean().optional(),
+    recap_email_enabled: z.boolean().optional(),
+    mobile_push_notifications_enabled: z.boolean().optional(),
+    mobile_push_discussion_activity_enabled: z.boolean().optional(),
+    mobile_push_admin_alerts_enabled: z.boolean().optional(),
+  }),
+  // TODO: Breaks `command(...)` type in `trpc.router`
```

Comment:
do you need a ticket to allow ZodEffect in commands?

Reply:
https://github.com/hicommonwealth/commonwealth/issues/7479